testing/windows_local_audio_client.py

```diff
@@ -0,0 +1,52 @@
+import os
+import sys
+import struct
+import numpy as np
+import time
+import requests
+import pyaudiowpatch as pyaudio
+
+def translator():
+    DOCKER_IP = "http://0.0.0.0:8000/"
+    print("TRANSLATOR IS RUNNING")
+
+    # open PyAudio manager via context manager
+    with pyaudio.PyAudio() as p:
+        # open audio stream via context manager
+        with p.open(format=pyaudio.paInt32, channels=2, rate=48000, input=True, frames_per_buffer=1024) as stream:
```

Comment:
audio settings change depending on the user's setup, it would be nice to not have these be hard-coded values

Also, the input device is never specified. Look at [the examples in the PyAudioWPatch repo](https://github.com/s0d3s/PyAudioWPatch/blob/master/examples/pawp_simple_recording_app.py) for how you are supposed to use this.